frontend/src/helpers/clearTokens.ts

```diff
@@ -0,0 +1,6 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+export const clearTokens = async () => {
+  await AsyncStorage.setItem('token', '');
+  await AsyncStorage.setItem('refreshToken', '');
```

Comment:
Use the setAsyncStorageItem function here. Or in case this function is unnecessary, remove it